includes/hiring_steps_default.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: plain
+---
+
+
+### Hiring Steps
```

Comment:
Is this something that was left over accidentally? Doesn't seem to have much sense and it's unused so better to remove